### PR TITLE
Broken links

### PR DESCRIPTION
**Pull request by jburel**
This PR attempts to reduce the number of links marked as "broken" or "ignored" in the daily doc build
e.g. https://ci.openmicroscopy.org/view/Failing/job/BIOFORMATS-DEV-latest-docs/

Check the output of 
https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-docs/
The job should be back to green

Still some suspicious links in the output but I have not found a good solution.
cc @sbesson @hflynn

### CODE REVIEW

File: docs/sphinx/formats/analyze-75.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ Analyze 7.5
 
 Extensions: .img, .hdr
 
-Developer: `Mayo Foundation Biomedical Imaging Resource <http://www.mayo.edu/bir>`_
+Developer: `Mayo Foundation Biomedical Imaging Resource <https://rportal.mayo.edu/bir/`_
```
Comment (sbesson) [inline docs/sphinx/formats/analyze-75.txt]: Is the trailing `>` replacement expected?

Comment (jburel) [inline docs/sphinx/formats/analyze-75.txt]: no well spotted

File: components/autogen/src/format-pages.txt [hunk 2]
```diff
@@ -1928,7 +1928,7 @@ API to read and write PNG files. \n
 [Prairie Technologies TIFF]
 pagename = prairie-tech-tiff
 extensions = .tif, .xml, .cfg
-developer = `Prairie Technologies <http://www.prairie-technologies.com/>`_
+developer = `Prairie Technologies <https://www.bruker.com/products/fluorescence-microscopes/ultima-multiphoton-microscopy/ultima-in-vitro/overview.html/>`_
```
Comment (sbesson) [inline components/autogen/src/format-pages.txt]: Extra `/` at the end of this link differs from the generated format page

File: docs/sphinx/formats/animated-png.txt [hunk 3]
```diff
@@ -25,7 +25,7 @@ Writer: APNGWriter (:bsd-writer:`Source Code <APNGWriter.java>`)
 Freely Available Software:
 
 - `Firefox 3+ <http://www.mozilla.com/firefox>`_ 
-- `Opera 9.5+ <http://www.opera.com/download>`_ 
+- `Opera 9.5+ <http://www.opera.com>`_
```
Comment (sbesson) [inline docs/sphinx/formats/animated-png.txt]: This change should be propagated back to `format-pages.txt`

**Comment by jburel** [comment]
https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-docs/576/ is green

**Comment by jburel** [comment]
@sbesson: maybe we leave this PR few more days,
will check the console output and see if I can fix few more links

**Comment by jburel** [comment]
if ``http://cbia.fi.muni.cz/`` is down for a period of time, we can always use the overarching link
`https://www.fi.muni.cz/research/laboratories/cbia.xhtml``

**Comment by jburel** [comment]
``http://cbia.fi.muni.cz/`` and ``http://www.nrao.edu/`` are both back

**Comment by sbesson** [comment]
The set of changes proposed in this PR make sense to me and both the auto-generated component and the documentation are updated. Assuming the docs build are green tomorrow, this should be ready to merge.

**Comment by jburel** [comment]
https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-docs/578/ is green 
and the number of links marked as ``broken`` is significantly reduced
Merging

**Comment by hflynn** [comment]
👍

**Pull request merged**